velox/serializers/PrestoSerializer.cpp

```diff
@@ -53,6 +53,7 @@ int64_t computeChecksum(
   auto remainingBytes = uncompressedSize;
   while (remainingBytes > 0) {
     auto data = source->nextView(remainingBytes);
+    VELOX_CHECK_GT(data.size(), 0);
```

Comment:
Can we add another check here:
Add a method ByteStream::remainingSize() and check remainingSize() == remainingBytes outside of the while loop? Thanks

Reply:
uncompressedSize is actually got from ByteStream::remainingSize()

Reply:
I think uncompressedSize is read from encoding of the stream.